en/faq.md

```diff
@@ -0,0 +1,36 @@
+# FAQ - Frequently Asked Questions
+
+## What is Phalcon
+Phalcon in an open source full stack PHP framework written as a C language extension.
+
+## How Phalcon Works
+Phalcon is *not* an accelerator for PHP or projects written in PHP. 
+
+Phalcon is a framework that implements its functionality using the low-level C language. C extensions are compiled together with your PHP code on load. Increasing the speed of you application and lowering your overhead.
+
+#### Phalcon achieves this by:
+
+- Taking advantage of native compilation by producing a [binary executable representation](http://en.wikipedia.org/wiki/Machine_code) of the code that a processor can directly understand and execute without the overhead of running bytecode in a virtual machine (VM).
+
+- Reducing the memory footprint by using optimised specific-purpose C structures and static types
+C compilers, like GCC/CLANG/VCC. These perform [several optimisations](http://en.wikipedia.org/wiki/Category:Compiler_optimizations) over the code improving performance.
+
+- The ability to place variables and data in the stack. These would typically have a higher [locality](http://en.wikipedia.org/wiki/Locality_of_reference) of access.
+
+- [Branch prediction](http://en.wikipedia.org/wiki/Branch_predictor) is easier as it operates directly over the user code and not over the VM implementation.
```

Comment:
It might be a good idea to use https://en.wikipedia.org/wiki/Predication_(computer_architecture)
Instead of http://en.wikipedia.org/wiki/Branch_predictor

The first one is about software and the one used is about the hardware version. While similar we should evaluate both

Reply:
What about this https://stackoverflow.com/a/11227902/1661465